contents/flood_fill/code/java/flood_fill.java

```diff
@@ -0,0 +1,97 @@
+package code.java;
+import java.util.*;
+public class flood_fill {
+        static class Point{
+            int x;
+            int y;
+            public Point(int x,int y){
+                this.x = x;
+                this.y = y;
+            }
+        }
+        public static boolean inbound(int[][] grid, Point p){
+            int row = grid.length, col = grid[0].length;
+            if (p.x < 0 || p.x >= row || p.y < 0 || p.y >= col) return false;
+            return true;
```

Comment:
maybe you can just do:
```suggestion
            return (p.x < 0 || p.x >= row || p.y < 0 || p.y >= col);
```

Reply:
sorry, wrong call: it was missing a `!` operator before the parenthesis...